### ISSUE TITLE
Theme.json: Core blocks should also be defined without the core keyword

### ISSUE THREAD

**Issue by MaggieCabrera** [WordPress/gutenberg#42298]
When editing theme.json and adding styles for blocks, they should be able to be defined without the need of prefixing them with `core/`

**Comment by annezazu** [comment]
@MaggieCabrera can you share a bit more why you want to see this/how this would be advantageous?

**Comment by MaggieCabrera** [comment]
> @MaggieCabrera can you share a bit more why you want to see this/how this would be advantageous?

yeah, I'm sorry for the one-liner! I was trying to track some discussions I had with @mtias and he suggested this so I opened the issue. I think it's mainly for convenience so you don't need to write `core/` every time, but both namings would work

**Comment by annezazu** [comment]
Got it! Thank you :)

**Comment by mtias** [comment]
Also for consistency with the HTML markup, where we allow core blocks to have the form of `wp:heading` and don't require `wp:core/heading`

**Comment by jasmussen** [comment]
@MaggieCabrera I'm coming back to this one, does it still feel valid or should we close here, 4 years later?

**Comment by MaggieCabrera** [comment]
> [@MaggieCabrera](https://github.com/MaggieCabrera) I'm coming back to this one, does it still feel valid or should we close here, 4 years later?

I think it's still valid, just not something we have prioritized
